iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -2011,7 +2051,7 @@ + (void)handleNotificationOpened:(NSDictionary*)messageDict
     [OneSignalHelper lastMessageReceived:messageDict];
     if (!foreground) {
         OneSignal.appEntryState = NOTIFICATION_CLICK;
-        [OneSignal.sessionManager onDirectSessionFromNotificationOpen:messageId];
+        [[OneSignal sessionManager] onDirectInfluenceFromNotificationOpen:_appEntryState withNotificationId:messageId];
```

Comment:
`OneSignal.sessionManager` is a getter and it is easier to read then `[OneSignal sessionManager]`. This is also close to Swift so it will be easier to migrate later.